Removes shipping related screenshots

Ferdev opened the pull request:
## Description
It also renames a few more WooCommerce Shipping & Tax entries

### Related issue(s)

Fixes #WOOSHIP-1272

### Steps to reproduce & screenshots/GIFs

N/A

### Checklist

<!-- All testable code should have tests. -->
- [ ] unit tests

<!-- An entry in the changelog file is always required -->
- [ ] `changelog.txt` entry added
- [ ] `readme.txt` entry added

Ferdev committed:
Removes shipping related screenshots

It also renames a few more WooCommerce Shipping & Tax entries

Ferdev committed:
Update changelog and readme

Ferdev commented:
Hey @allie500 , since you're the janitor and will release this, and it's a pretty simple PR, I just assigned you as reviewer directly, hope you don't mind :smile:

allie500 reviewed (changes requested):
Thanks for taking care of this, Fer! Changes look good. I found a couple of other references to _WooCommerce Shipping & Tax_. Most are not user facing. Things like method doc blocks, etc. But I do see it still exists in the [main README file](https://github.com/Automattic/woocommerce-services/blob/fix/WOOSHIP-1272/remove-shipping-screenshots/README.md#L1). I think we should update that file in this PR.

Below are a few other files that we may want to update as well since they are dev and possibly user facing. 

- [Store on WPCOM README](https://github.com/Automattic/woocommerce-services/blob/fix/WOOSHIP-1272/remove-shipping-screenshots/client/extensions/woocommerce/README.md#L12)
- [Contributing doc](https://github.com/Automattic/woocommerce-services/blob/fix/WOOSHIP-1272/remove-shipping-screenshots/CONTRIBUTING.md#L1)
- [Activity Log README](https://github.com/Automattic/woocommerce-services/blob/fix/WOOSHIP-1272/remove-shipping-screenshots/client/extensions/woocommerce/state/sites/orders/activity-log/README.md#L6)

I don't think these are a blocker though. Just the main README should be updated at minimum.

Ferdev committed:
Rename all remaining WooCommerce Shipping & Tax entries

Mainly from docs and code comments

Ferdev commented:
Sounds good Allie, thanks! I updated all remaining entries of WooCommerce Shipping & Tax that I could find. I hadn't removed them because they were old code comments mainly, but you're right is confusing so I just went ahead and updated everything. Thanks for the attention to detail :bow:

allie500 reviewed (approved):
Thanks for addressing the user facing references of the old plugin name as well as the additional updates in the README to remove references to things like shipping label printing. Also, kudos for taking care of the old plugin references in the function doc blocks. I think this is ready to ship now.
